### ISSUE TITLE
Restarting Thunderbird doesn't restart the theme

### ISSUE THREAD

**Issue by Falcon1208** [conema/monterail-fulldark#59]
Hi,

Each time I restart Thunderbird  the theme isn't here. I must un-active and re-active it.

78.5.0 (32 bits), windows 10

**Comment by conema** [comment]
Hi @Falcon1208,
I cannot reproduce this problem.

 Could you try to uninstall the theme, exit from thunderbird (completely) and reinstall the theme? 

Do you have the same problem with the dark version? (https://addons.thunderbird.net/en-US/thunderbird/addon/monterail-dark-2-0-for-tb-68/)

**Comment by Falcon1208** [comment]
Hello Conema,

I uninstalled the theme, exit from thunderbird, and restart. 
After that I downloaded monterail_dark_2_for_tb68-3.1.0-tb.xpi and install it.

Same problem. And now a new problem  : the background color form of Thunderbird's main form is white ! :( :( :( my eyes are suffering again.

**Comment by conema** [comment]
> the background color form of Thunderbird's main form is white ! :( :( :( my eyes are suffering again.

Yes, the "dark" version is lighter than the "full dark", you can reinstall the full dark one.

Do you have other plugins installed? Did you try to re-install thunderbird?
